src/test/unit/controls/Gantt/Track-spec.js

```diff
@@ -735,4 +735,21 @@ describe("Gantt -> Track", () => {
             });
         });
     });
+    describe("Gantt chart div element for the label popup tooltip ", () => {
+        describe("when chart is loaded", () => {
+            it("add div element for the popup tooltip", () => {
+                expect(
+                    fetchElementByClass(styles.labelPopupTooltip)
+                ).not.toBeNull();
+            });
+        });
+        describe("when chart is destroyed", () => {
+            it("remove div element for the popup tooltip", () => {
```

Comment:
Grammar nit: add -> adds, remove -> removes

Reply:
> Grammar nit: add -> adds, remove -> removes

[fixed](https://github.com/cerner/carbon-graphs/pull/272/commits/11a0be4eaf177602301435950dc6b3ff816881b0#diff-11aa0aaf6db553889f349765e88f7462R2000-R2012)

Reply:
Nit: I think you missed this file.
add -> adds, remove -> removes

Reply:
[fixed](https://github.com/cerner/carbon-graphs/pull/272/commits/91e81c529c41dadbf5a2a203614d91e9413226b0#diff-7fb4a16b40433875e7e73811189b4f84R738-R750)